storage/tianmu/handler/ha_tianmu.h

```diff
@@ -171,7 +171,7 @@ class ha_tianmu final : public handler {
   core::TableID tmp_table_;
   std::unique_ptr<core::CompiledQuery> cq_;
```

Comment:
some unique_ptr varible named with suffix  _ptr_, while some variables named without the suffix. Better to unform it.